tests/tls/ca.crt

```diff
@@ -0,0 +1,31 @@
+-----BEGIN CERTIFICATE-----
```

Comment:
Why do you include the certs in the repo? For tests it doesn't really matter but it's not a good practice, can trigger false positive security alerts for the repo - and besides we generate them anyway.

Reply:
included as a lasy local run ( so that I didnt had to specify their paths :) ). Will remove them :+1: